### PR TITLE
lint.go: Improve output of lint help command

### LINKED ISSUE #42: "help lint" response should be updated.
Present output of Configlet to command `help lint` as v3.2.0.
```
Verify that the config.json file is valid, and that the exercises are complete.

Call lint command with path to track:

  configlet lint path/to/track

Usage:
  configlet lint [flags]

Flags:
  -h, --help   help for lint
```
As of v3.2.0 `lint` is now verifying that `config/maintainers.json` is valid JSON.  Should the help response from configlet convey this?
```
Verify that the config.json file is valid, and that the exercises are complete.
Verify that the maintainers.json is valid JSON.

Call lint command with path to track:

  configlet lint path/to/track

Usage:
  configlet lint [flags]

Flags:
  -h, --help   help for lint
```

### PR DESCRIPTION
**Pull request by robphoenix**
Closes #42

### CODE REVIEW

**Comment by robphoenix** [comment]
Example of output:

```
The lint command checks for any discrepancies in a track's configuration files.

It ensures the following files are valid JSON:
	config.json, maintainers.json

It also checks that the exercises defined in the config.json file are complete.

Usage:
  configlet lint <path/to/track>
```

File: cmd/lint.go [hunk 1]
```diff
@@ -8,26 +8,27 @@ import (
...
+}
+
+// lintUsageText defines usage of the lint command
+var lintUsageText = "Usage:\n  configlet lint <path/to/track>\n"
 
 func lint(cmd *cobra.Command, args []string) {
 	if len(args) == 0 {
 		lintUsageFunc(cmd)
-		os.Exit(1)
+		return
```
Comment (robphoenix) [inline cmd/lint.go:31]: I changed this so as to have a cleaner output.

From this: 

```
Usage:
  configlet lint <path/to/track>
exit status 1
```

To this:

```
Usage:
  configlet lint <path/to/track>
```

Comment (kytrinyx) [reply cmd/lint.go:31]: Much nicer.

**Review by kytrinyx — APPROVED** [review]
This looks great!

**Pull request merged**